src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -507,20 +506,16 @@ def _calibrate(self, module):
             if self.num_calibration_steps is None
             else cycle(self._calibration_dataloader)
         )
-        with suppress(StopIteration):
-            batch = next(_dataloader)
-            batch_number = 1
-            done = False
-            while not done:
-                batch = tensors_to_device(batch, model_device)
-                with torch.no_grad():
-                    forward_fn(batch, module=module)
-                done = (
-                    self.num_calibration_steps is not None
-                    and batch_number >= self.num_calibration_steps
-                )
-                batch = next(_dataloader)
-                batch_number += 1
+
+        for batch_idx, batch in enumerate(_dataloader):
+            _calibration_completed = (
```

Comment:
nit: put directly into if statement

Reply:
Any particular reason why we should prefer that over this?

Reply:
if there's no reason to put it into a separate variable, we shouldn't break it out

Reply:
My opinion is that it serves as a explanatory variable, describing why we are breaking out early, However will make this change if you insist.